api/v1/db/oracledb/game-dao.js

```diff
@@ -88,4 +88,28 @@ const getGameById = async (id) => {
   }
 };
 
-module.exports = { getGames, getGameById, getGamesByMemberId };
+/**
+ * @summary Check the if the game with the id exists.
+ * @function
+ * @param {Int} id The id of the game
+ * @returns {Bool} If the game exists or not.
```

Comment:
The return type here should be `Promise<boolean>` since the function is marked `async`